speechbrain/dataio/dataio.py

```diff
@@ -41,6 +43,9 @@ def load_data_json(json_path, replacements={}):
     replacements : dict
         (Optional dict), e.g., {"data_folder": "/home/speechbrain/data"}.
         This is used to recursively format all string values in the data.
```

Comment:
I know this is not new in the PR but while the way replacements are used is hinted at in the example, it's never actually explained in text. IMO it is also unclear what recursion meant (whether it was somehow related to the replacement or the data structure being transformed).

```suggestion
    replacements : dict
        (Optional dict), e.g., {"data_folder": "/home/speechbrain/data"}.
        Replacements will replace text of the form `{key}` with the
        associated value.
        Replacements are applied recursively to all nested values in the
        input. This excludes keys in dictionaries.
```